upstream-community-operators/rook/rook.v0.1.0.csv.yaml

```diff
@@ -0,0 +1,305 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
filename must be `<...>.clusterserviceversion.yaml` for it to be picked up by operatorhub.